diff --git a/data.py b/data.py
index 43317a9..01bf76c 100644
--- a/data.py
+++ b/data.py
@@ -17,8 +17,8 @@ SHAPES = {
 
 def get_scatter_transform(dataset):
     shape = SHAPES[dataset]
-    scattering = Scattering2D(J=2, shape=shape[:2])
-    K = 81 * shape[2]
+    scattering = Scattering2D(J=2, L=11, shape=shape[:2])
+    K = 144 * shape[2]
     (h, w) = shape[:2]
     return scattering, K, (h//4, w//4)
 
